src/pyupgrade/plugins/os_error_alias.rs

```diff
@@ -0,0 +1,232 @@
+use itertools::Itertools;
+use rustpython_ast::{Excepthandler, ExcepthandlerKind, ExprKind, Located};
+
+use crate::ast::helpers::match_module_member;
+use crate::ast::types::Range;
+use crate::autofix::Fix;
+use crate::checkers::ast::Checker;
+use crate::checks::{Check, CheckKind};
+
+const ERROR_NAMES: &'static [&'static str] = &["EnvironmentError", "IOError", "WindowsError"];
+const ERROR_MODULES: &'static [&'static str] = &["mmap", "select", "socket"];
+
+fn get_correct_name(original: &str) -> String {
+    if ERROR_NAMES.contains(&original) {
+        return "OSError".to_string();
+    } else {
+        return original.to_string();
+    }
+}
+
+fn get_before_replace(elts: &Vec<Located<ExprKind>>) -> Vec<String> {
+    elts.iter()
+        .map(|elt| {
+            if let ExprKind::Name { id, .. } = &elt.node {
+                id.to_string()
+            } else {
+                "".to_string()
+            }
+        })
+        .collect()
+}
+
+fn check_module(checker: &Checker, expr: &Located<ExprKind>) -> (Vec<String>, Vec<String>) {
```

Comment:
Everywhere that we do `Located<ExprKind>`, we should instead of the RustPython type `Expr`. There are analogous types for every `Located<T>`.

Reply:
This is very good to know.